awx/ui/src/screens/Instances/Shared/InstanceForm.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { t } from '@lingui/macro';
+import { Formik, useField } from 'formik';
+import { Form, FormGroup, CardBody } from '@patternfly/react-core';
+import { FormColumnLayout } from 'components/FormLayout';
+import FormField, { FormSubmitError } from 'components/FormField';
+import FormActionGroup from 'components/FormActionGroup';
+import { required } from 'util/validators';
+import AnsibleSelect from 'components/AnsibleSelect';
+import {
+  ExecutionEnvironmentLookup,
+  InstanceGroupsLookup,
+} from 'components/Lookup';
+
+// This is hard coded because the API does not have the ability to send us a list that contains
+// only the types of instances that can be added.  Control and Hybrid instances cannot be added.
+
+const INSTANCE_TYPES = [
+  { id: 2, name: t`Execution`, value: 'execution' },
```

Comment:
`id` here is used by `AnsibleSelect` to set the element id. We should make this a string value and not an integer.